.github/workflows/assets.yml

```diff
@@ -55,6 +55,10 @@ jobs:
           rm -rf assets && mkdir assets && cd assets
           docker run "$EVE" rootfs > rootfs.img
           docker run "$EVE" installer_net | tar -xvf -
+          if [ "${{ env.ARCH }}" = amd64 ]; then
```

Comment:
Is nupkg not possible for ARM?

Reply:
The problem here is inside name of the file. During experiments with nuget and local files (not published into gallery) I noticed that we can use only one file name notation. So I thing it is better to have amd64 version here as more common for Windows, than not working (with requirement to rename the file).